### PR TITLE
Update to new docs generator

### PR DESCRIPTION
**Pull request by compleatang**

### CODE REVIEW

File: docs/tests/contracts_deploying_adv/test.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+chains_dir=$HOME/.eris/chains
+chain_name=advchain2
+name_full="$chain_name"_full_000
+name_part="$chain_name"_participant_000
+chain_dir=$chains_dir/$chain_name
+eris chains make --account-types=Full:1,Participant:1 $chain_name
+key1_addr=$(cat $chain_dir/addresses.csv | grep $name_full | cut -d ',' -f 1)
+eris chains new $chain_name --dir $chain_dir/$name_full 1>/dev/null
```
Comment (VoR0220) [inline docs/tests/contracts_deploying_adv/test.sh]: This needs to be start when we drop 0.12.0-rc3. But that's a nitpick.

**Comment by VoR0220** [comment]
Not sure what you want to do about that little bit, if we should hold off to merge this, or should we do it right away?

**Comment by VoR0220** [comment]
other than that....looks great. Excited.

**Comment by compleatang** [comment]
merge this in. the start|new for chains isn't even on develop.

**Pull request merged**